pyproject.toml

```diff
@@ -98,6 +98,7 @@ gpt4all = { version = "^1.0.8", optional = true }
 elasticsearch = { version = "^8.9.0", optional = true }
 flask = "^2.3.3"
 twilio = "^8.5.0"
+wikipedia = "^1.4.0"
```

Comment:
We have to decide what dependencies we want to include in the core package. This is an ongoing discussion